src/main/java/com/atlauncher/managers/CurseForgeUpdateManager.java

```diff
@@ -69,30 +67,11 @@ public static void checkForUpdates() {
                                 .findFirst().orElse(null);
 
                         if (latestVersion == null) {
-                            return false;
-                        }
-
-                        // if there is a change to the latestversion for an instance (but not a first
-                        // time write), then refresh instances panel
-                        if (Data.CURSEFORGE_INSTANCE_LATEST_VERSION.containsKey(i)
-                                && Data.CURSEFORGE_INSTANCE_LATEST_VERSION.get(i).id != latestVersion.id) {
-                            wasUpdated = true;
-                        }
-
-                        // updated if there is no latest version stored yet but the instance has update
-                        if (!Data.CURSEFORGE_INSTANCE_LATEST_VERSION.containsKey(i)
-                                && latestVersion.id != i.launcher.curseForgeFile.id) {
-                            wasUpdated = true;
+                            return;
                         }
 
                         Data.CURSEFORGE_INSTANCE_LATEST_VERSION.put(i, latestVersion);
-
-                        return wasUpdated;
-                    }).anyMatch(b -> b);
-
-            if (refreshInstancesPanel) {
-                App.launcher.reloadInstancesPanel();
```

Comment:
So where does this get triggered now when these find an update?

Reply:
Good catch, I'll reanalyze the stack before this PR is ready